xbmc/cores/AudioEngine/AEFactory.cpp

```diff
@@ -273,6 +273,11 @@ void CAEFactory::Shutdown()
 IAEStream *CAEFactory::MakeStream(enum AEDataFormat dataFormat, unsigned int sampleRate, 
   unsigned int encodedSampleRate, CAEChannelInfo channelLayout, unsigned int options)
 {
+  if (dataFormat == AE_FMT_INVALID || dataFormat == AE_FMT_MAX)
```

Comment:
If it comes in, maybe better then to use with ```dataFormat >= AE_FMT_MAX```to prevent values out of limit (I know normally never, but why not ;) )